ibctest/test_chains.go

```diff
@@ -0,0 +1,39 @@
+package ibctest
+
+import (
+	"fmt"
+
+	"github.com/strangelove-ventures/ibc-test-framework/chain/cosmos"
+	"github.com/strangelove-ventures/ibc-test-framework/ibc"
+)
+
+var chainConfigs = []ibc.ChainConfig{
+	cosmos.NewCosmosHeighlinerChainConfig("gaia", "gaiad", "cosmos", "uatom", "0.01uatom", 1.3, "504h"),
+	cosmos.NewCosmosHeighlinerChainConfig("osmosis", "osmosisd", "osmo", "uosmo", "0.0uosmo", 1.3, "336h"),
+	cosmos.NewCosmosHeighlinerChainConfig("juno", "junod", "juno", "ujuno", "0.0025ujuno", 1.3, "672h"),
+}
+
+var chainConfigMap map[string]ibc.ChainConfig
+
+func init() {
+	chainConfigMap = make(map[string]ibc.ChainConfig)
+	for _, chainConfig := range chainConfigs {
+		chainConfigMap[chainConfig.Name] = chainConfig
+	}
+}
+
+func GetChain(testName, name, version, chainID string, numValidators, numFullNodes int) (ibc.Chain, error) {
+	chainConfig, exists := chainConfigMap[name]
+	if !exists {
+		return nil, fmt.Errorf("No chain configuration for %s", name)
```

Comment:
Error strings should begin with a lowercase letter.

Reply:
A linter can catch this in the future.

Reply:
the `golangci-lint` linter we are using catches the errors.New() capital first letters but not the fmt.Errorf. Maybe we can configure it to, or maybe we should change the linter or add an additional one?

Reply:
https://staticcheck.io/ has been the canonical linter in my experience.